### ISSUE TITLE
Allow for rScaleType to be sent with parameters

### ISSUE THREAD

**Issue by susielu** [emeeks/semiotic#353]
I tried making a pull request for this and failed miserably 😭

**Comment by susielu** [comment]
Goal is to be able to send something like this: 
```

const frameProps = {
  rExtent: [0],
  size: [300, 100],
  projection: "horizontal",
  rAccessor: "key",
  rScaleType: scalePow()
    .exponent(0.5),
  oAccessor: "name",
  margin: { bottom: 10 },
  summaryType: {
    type: "histogram",
    amplitude: 10,
    binValue: d => d[0].doc_count,
    useBins: false
  }
}

```

**Comment by emeeks** [comment]
Available in `1.15.9`
